using-mobile-extensions/mobile-core/README.md

```diff
@@ -1,34 +1,34 @@
 # Mobile Core
 
-The Mobile Core represents the core Adobe Experience Platform SDK that is required for every app implementation. The core contains a common set of functionality and frameworks, such as Experience Cloud Identity services, data event hub, Rules Engine, reusable networking, disk access routines, and so on, which are required by all Adobe and third-party extensions. The following sections provide additional information about what is provided in the Mobile Core extension.
+Mobile Core is a mandatory Adobe Experience Platform SDK for app implementation. Mobile Core contains a common set of functionality and frameworks, including Experience Cloud Identity services, data event hub, Rules Engine, reusable networking, and disk access routines. The following sections provide additional information about what is provided in the Mobile Core extension.
 
 ## Mobile Core generic APIs
 
 The SDK provides core, extension-independent APIs to facilitate the event tracking of user screens, actions, and PII data. Events generated by these APIs are published to the SDK event hub and are available for use by extensions. For instance, when the Analytics extension is installed, all user actions and app screens event data is sent to the appropriate Analytics reporting endpoints.
 
-For more information, see [Mobile Core API Reference](mobile-core-api-reference.md).
+For more information, please read the [Mobile Core API reference](mobile-core-api-reference.md).
 
 ## Configuration
 
-The configuration extension provides the events in the Event Hub that provide the configuration details for all installed extensions. For more information, see [Configuration](configuration/).
+The configuration extension provides different APIs to set up your application's configuration settings. For more information, please read the documentation on [configuration](configuration/).
```

Comment:
Double-check with Jiaben Geng that this is okay. Posted in the doc slack for his verification.

Reply:
Jiabin says it looks good.